magefiles/utils.go

```diff
@@ -154,3 +156,65 @@ func renderTemplate(destination, templatePath string, templateData interface{},
 
 	return nil
 }
+
+func cleanupQuay(quayService quay.QuayService, quayOrg string) error {
+
+	r, err := regexp.Compile(`^(e2e-demos|has-e2e)`)
```

Comment:
please add the repo prefix to a constant. it could change in a future and it's used more than once in this function